libs/data-types/src/lib/decorators/FixedValue.ts

```diff
@@ -0,0 +1,17 @@
+import { Equals } from 'class-validator';
+import mixinDefaultTypeDecoratorOptions from './common/mixinDefaultTypeDecoratorOptions';
+import { TypeDecoratorOptions } from './types/TypeDecoratorOptions';
+import WithValidation from './validation/WithValidation';
+
+export function FixedValue(
+    value: string | number | boolean,
+    userOptions: TypeDecoratorOptions
+): PropertyDecorator {
+    return (target: Object, propertyKey: string | symbol) => {
+        const options = mixinDefaultTypeDecoratorOptions(userOptions);
+
+        WithValidation(Equals(value), options)(target, propertyKey);
+
+        // appendMetadata(target, propertyKey, 'REMOVE ME', options);
```

Comment:
remove commented out code

Reply:
Ah- the point is to remove the entire file (meant to be caught here), but you wouldn't have known that.

Reply:
I've removed the file now.